tests/test_charging_temperature.py

```diff
@@ -0,0 +1,34 @@
+from unittest import TestCase
+
+from src.check_limits import BatteryManagementSystem
+
+
+class TestChargingTemperature(TestCase):
+
+    def setUp(self) -> None:
+        self.battery_obj = BatteryManagementSystem()
+
+    def test_low_temperature_scenario_en(self):
+        self.battery_obj.set_charging_temperature(-1)
+        self.assertRaises(ValueError, self.battery_obj.verify_charging_temperature_range_en)
+
+    def test_low_temperature_scenario_de(self):
+        self.battery_obj.set_charging_temperature(-1)
+        self.assertRaises(ValueError, self.battery_obj.verify_charging_temperature_range_de)
+
+    def test_high_temperature_scenario_en(self):
+        self.battery_obj.set_charging_temperature(70)
+        self.assertRaises(ValueError, self.battery_obj.verify_charging_temperature_range_en)
```

Comment:
how do we verify if proper language strings are printed? it is good to add test case and make code testable.